### DISCUSSION TITLE
BodyElement ausschalten?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2568] [General]
Im Gerät ist kein BodyElement ausgewählt und trotzdem wird es im Layout angezeigt.
Wie kann man es löschen? Es gibt kein X.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/414bc705-7d16-44a4-891f-4bc0ee70552b)

**Comment by Zefau** [comment]
Auswählen und dann löschen.

**Comment by mcuiobroker** [comment]
Wäre es nicht sinnvoll es gar nicht anzuzeigen, wenn es im Gerät nicht ausgewählt wurde?
